style.css

```diff
@@ -806,141 +345,64 @@
 .about {
   display: flex;
   flex-direction: column;
-  align-items: center;
-  padding: 64px 0;
-  position: absolute;
-  width: 375px;
-  height: 1625px;
-  left: 0;
-  top: 4050px;
-  margin: 12px 0;
 }
 
 .contact-section {
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 64px 0;
-  position: absolute;
-  width: 375px;
-  height: 833px;
-  left: 0;
-  top: 5749px;
+  width: auto;
   margin: 12px 0;
-  background: url(icons/Frame.png) right bottom no-repeat;
+  background: url(/icons/Frame.png) right bottom no-repeat;
 }
 
-.contact-header {
+.contact-header h2 {
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 0;
-  position: static;
   width: 327px;
-  height: 363px;
-  left: 24px;
-  top: calc(50% - 363px / 2 - 194px);
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin: 58px 0;
-}
-
-.contact-header h2 {
-  position: static;
-  width: 327px;
-  height: 363px;
-  left: 0;
-  top: 0;
   font-family: 'Inter', serif, sans-serif;
   font-style: normal;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
   text-align: center;
   color: #172b4d;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin: 12px 0;
 }
 
 .contact-form {
   display: flex;
   flex-direction: column;
```

Comment:
- [OPTIONAL ] What do you think to add more `padding-top`?

- Your design:

![033333](https://user-images.githubusercontent.com/79058364/161379189-97687a47-4054-468b-9107-fbf77acd985a.png)

- figma design:

![go111111](https://user-images.githubusercontent.com/79058364/161379300-a25ab37c-7df2-4dce-a992-2f77502fafa4.png)